app/libs/file.js

```diff
@@ -28,6 +30,9 @@ function getAllTracksFileId(musicFiles) {
             }
             saveTrack(musicFiles, tracksFileId);
             deleteTrack(allTracksFiles, musicFilesFileId);
+            const msg = 'All your audio files have been added';
+            application.appLayout.getRegion('toolbar')
+                .currentView.showNotification(msg);
```

Comment:
Here's a glitch: you should never call a view outside of the view tree, or you break the separation of concerns. So, instead of _explicitely_ call the view here, you should pass an event in the [`application.channel` ](http://devdocs.io/marionette/marionette.application#the-application-channel) and then the application or (better) the Notification top component will decide itself to react to this event.